recipes/genesis2/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "genesis2" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/Genesis-1.3-Version2/archive/v{{ version }}.tar.gz
+  sha256: 0c8be1fd80cfa8339e3e40b5e798ec596d339e9b743c46a187321a71b9766e12
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('fortran') }}
```

Would suggest using MSYS2's Fortran compiler on Windows like so

```suggestion
    - {{ compiler('fortran') }}              # [not win]
    - {{ compiler('m2w64_fortran') }}        # [win]
```